datadog-agent-nvml.yaml

```diff
@@ -0,0 +1,102 @@
+
+  - runs: |
+      # Create and activate a virtual environment.
+      python -m venv .venv
+      . .venv/bin/activate
+
+      # Use embedded Python in virtual environment.
+      sed -i "s|$(pwd)/.venv|${{vars.dd_shared}}|g" .venv/pyvenv.cfg
+      sed -i "s|$(pwd)/.venv|${{vars.dd_shared}}|g" .venv/bin/*
+
+      # Dump the packages.
+      mkdir site-packages
+      rsync -az ${{vars.dd_shared}}/lib/python${{vars.python_version}}/site-packages/ ./site-packages/
+
+      # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
+      SOURCE_DATE_EPOCH=315532800
+
+      # Install integration.
+      pip install ./nvml
+
+      # Cleanup before preparing the package content.
+      find ${{vars.dd_shared}} -name "*.pyc" -delete
+      find ${{vars.dd_shared}} -name "__pycache__" -exec rm -rf {} +
+
+      # Prepare the package content.
+      diff_packages=$(rsync --include='*/' --exclude='*' --ignore-existing -rv --dry-run ${{vars.dd_shared}}/lib/python${{vars.python_version}}/site-packages/ ./site-packages/ | grep -v "sending incremental file list" | grep -v -E "^\./$|^sent .+ bytes|total size|^$")
+
+      # Include the new packages.
+      packages_dir="${{targets.contextdir}}${{vars.dd_shared}}/lib/python${{vars.python_version}}/site-packages/"
+      mkdir -p "${packages_dir}/datadog_checks"
+      for package in $diff_packages; do
+        rsync -avz "${{vars.dd_shared}}/lib/python${{vars.python_version}}/site-packages/${package}" "${packages_dir}/${package}";
+      done
+
+      # Include the new configurations.
+      conf_dir="${{targets.destdir}}${{vars.dd_conf}}/nvml.d"
+      mkdir -p $conf_dir
+      rsync -avz ${{vars.dd_shared}}/lib/python${{vars.python_version}}/site-packages/datadog_checks/nvml/data/ $conf_dir/
+
+      deactivate
+
+test:
+  environment:
+    contents:
+      packages:
+        - datadog-agent
+        - datadog-agent-core-integrations
+        - datadog-agent-nvml=${{package.full-version}}
+    environment:
+      PYTHONPATH: /usr/share/datadog-agent/lib/python${{vars.python_version}}/site-packages
+      PATH: /opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/lib/jvm/java-11-openjdk/bi
+  pipeline:
+    - runs: |
+        ls /etc/datadog-agent/conf.d/nvml.d/conf.yaml.example
+        ls /usr/share/datadog-agent/lib/python${{vars.python_version}}/site-packages/datadog_checks/nvml/__init__.py
+        ls /usr/share/datadog-agent/lib/python${{vars.python_version}}/site-packages/datadog_nvml-${{package.version}}.dist-info/WHEEL
+    - name: Verify the integration is installed # https://docs.datadoghq.com/agent/guide/integration-management/?tab=linux#install
+      runs: |
+        cp /opt/datadog-agent/requirements-agent-release.txt ${{vars.dd_home}}/
+        cp /opt/datadog-agent/final_constraints-py3.txt ${{vars.dd_home}}/
```

Comment:
Do we need to do this in packaging?

Reply:
Yes, thanks for the catch. [Indeed](https://github.com/DataDog/datadog-agent/blob/6f52a65f70bd94220d49e8b7434c2b162dab30be/cmd/agent/subcommands/integrations/command.go#L385)

Reply:
Addressed @EyeCantCU in [this commit](https://github.com/wolfi-dev/os/pull/31578/commits/f6809eff0926b184421f32153fd9f46eeca9cc1a), PTAL.